### ISSUE TITLE
SD 2.1 training is not working as expected

### ISSUE THREAD

**Issue by terrychahn** [google/maxdiffusion#73]
When using the base21.yml script to train a SD 2.1 model, the generated checkpoint does not appear to be trained from the dataset correctly. The output images are the same as the base model.
It has also the same output after training with modified params of 15000 training steps and a learning rate of 1e-5.

**Comment by entrpn** [comment]
Fixed through #70 - mainly keeping weights in fp32. Closing this issue, but feel free to re-open.
